README.md

```diff
@@ -99,19 +89,14 @@ After you're finished please remove all the comments and instructions!
 
 > Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- Added HTML file
+- Added CSS file
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
+> coming soon
 
```

Comment:
- [ ] This is looking like a standalone without any information above kindly remove it so your readme looks professional :+1: